reana/cli.py

```diff
@@ -627,6 +627,20 @@ def is_component_runnable_example(component):
     return False
 
 
+def does_component_need_db(component):
```

Comment:
Can you add a tiny test case for this new function, similarly to what we have for others?